xla/stream_executor/stream.h

```diff
@@ -881,11 +881,13 @@ class Stream {
                            const DeviceMemory<InputType> &a, int lda,
                            const DeviceMemory<InputType> &b, int ldb,
                            DeviceMemory<OutputType> *c, int ldc,
-                           const NumericOptions &numeric_options) {
+                           const NumericOptions &numeric_options,
+                           blas::CallContext context) {
```

Comment:
These also need defaults to avoid API breakage.

Reply:
Done